### ISSUE TITLE
Address Contrast Failures in sort buttons

### ISSUE THREAD

**Issue by jenndiaz** [wwcodecolorado/scrollodex#94]
#### Description

The sort buttons at the top of the page, `Initial`, `Alphabetical` and `Randomize` do not meet contrast minimums. 

#### Files

[Button implementation](https://github.com/wwcodecolorado/scrollodex/blob/main/src/cardsDisplay.jsx#L98) 
[scss for buttons](https://github.com/wwcodecolorado/scrollodex/blob/main/src/Components/Buttons.scss)

#### Screenshots
![contrast issues](https://user-images.githubusercontent.com/63808889/194717825-21d0a2a7-cfc5-43d4-ac89-bf2688de8333.jpg)

#### Tasks

- [x] Change styling of these button to meet contrast guidelines and match new style guide 

#### Notes: 

Women Who Code Teal is #0077D7F

Check out [RESOURCES.md](https://github.com/wwcodecolorado/scrollodex/blob/main/RESOURCES.md) for more accessibility resources and how to use the Wave Extension on the codebase.

**Comment by ankeshp03** [comment]
Hi, may I work on this issue?
